accepted/2021/default_string_globalization.md

```diff
@@ -0,0 +1,390 @@
+     public System.String Trim(char trimChar) { throw null; }
+     public System.String Trim(params char[]? trimChars) { throw null; }
+     public System.String TrimEnd() { throw null; }
+     public System.String TrimEnd(char trimChar) { throw null; }
+     public System.String TrimEnd(params char[]? trimChars) { throw null; }
+     public System.String TrimStart() { throw null; }
+     public System.String TrimStart(char trimChar) { throw null; }
+     public System.String TrimStart(params char[]? trimChars) { throw null; }
+ }
+ public abstract partial class StringComparer : System.Collections.Generic.IComparer<string?>, System.Collections.Generic. IEqualityComparer<string?>, System.Collections.IComparer, System.Collections.IEqualityComparer
+ {
+     protected StringComparer() { }
+-    public static System.StringComparer CurrentCulture { get { throw null; } }
+-    public static System.StringComparer CurrentCultureIgnoreCase { get { throw null; } }
+-    public static System.StringComparer InvariantCulture { get { throw null; } }
+-    public static System.StringComparer InvariantCultureIgnoreCase { get { throw null; } }
+-    public static System.StringComparer Ordinal { get { throw null; } }
+-    public static System.StringComparer OrdinalIgnoreCase { get { throw null; } }
++    public static System.StringComparer Normal { get { throw null; } } // aliases Ordinal
++    public static System.StringComparer IgnoreCase { get { throw null; } } // aliases OrdinalIgnoreCase
+     public int Compare(object? x, object? y) { throw null; }
+     public abstract int Compare(string? x, string? y);
+-    public static System.StringComparer Create(System.Globalization.CultureInfo culture, bool ignoreCase) { throw null; }
+-    public static System.StringComparer Create(System.Globalization.CultureInfo culture, System.Globalization.CompareOptions options) { throw null; }
+     public new bool Equals(object? x, object? y) { throw null; }
+     public abstract bool Equals(string? x, string? y);
+     public static System.StringComparer FromComparison(System.StringComparison comparisonType) { throw null; }
+     public int GetHashCode(object obj) { throw null; }
+     public abstract int GetHashCode(string obj);
+ }
+ public enum StringComparison
+ {
+-    CurrentCulture = 0,
+-    CurrentCultureIgnoreCase = 1,
+-    InvariantCulture = 2,
+-    InvariantCultureIgnoreCase = 3,
+-    Ordinal = 4,
+-    OrdinalIgnoreCase = 5,
++    Normal = 4, // aliases Ordinal
++    IgnoreCase = 5, // aliases OrdinalIgnoreCase
+ }
+```
+
+## A note on `StringComparison` and `StringComparer`
+
+One notable aspect of this proposal is the complete restructuring of the `StringComparison` and `StringComparer` types. It is important to note that all members of these types will be preserved. The proposal here is to hide all existing fields / static properties and to create new "simpler" names _Normal_ and _IgnoreCase_. Both of these additions alias their ordinal siblings.
```

Comment:
May be it is worth mentioning whoever want to create a linguistic StringComparer, can call the extension method `CompareInfo.GetStringComparer`. This will be explicit as it will be created from the CompareInfo which show the intend of using the linguistic functionality.